playwright-e2e/dss/component/select.ts

```diff
@@ -100,4 +100,12 @@ export default class Select extends WidgetBase {
     }
     return options!;
   }
+
+  get currentValue(): Promise<string> {
+    return this.toLocator().locator('span.mat-select-min-line').textContent() as Promise<string>;
+  }
```

Comment:
Certainly, Alex, it can indeed be rewritten, and the choice largely hinges on personal preferences and established coding patterns. 
If you find it more readable, I'd be happy to make the adjustment.

Reply:
Done!